app/models/comment.rb

```diff
@@ -1,2 +1,7 @@
 class Comment < ApplicationRecord
+  validates :text, length: { minimum: 1 }, presence: true
+  validates :post_id, presence: true
+  validates :user_id, presence: true
+  belongs_to :user
+  belongs_to :post
```

Comment:
Since you have established the association between comment and user tables and comment and post tables, a comment cannot be created unless a user_id and post_id is created. So validations in line 3 and 4 are not required. Please remove them.